src/eredis_cluster.erl

```diff
@@ -101,6 +101,50 @@ q(Command) ->
 qk(Command, PoolKey) ->
     query(Command, PoolKey).
 
+%% =============================================================================
+%% @doc Multi node query
+%% @end
+%% =============================================================================
+-spec qmn(redis_pipeline_command()) -> redis_pipeline_result().
+qmn(Commands) ->
+    CommandsByPools = split_by_pools(Commands),
```

Comment:
I've tried two incr and they work, anyway if we can increase performance that must be done, I will try to optimize request